### PR TITLE
KYLIN-3356 Constant in SecretKeySpec

### PR DESCRIPTION
**Pull request by GinaZhai**

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

File: core-common/src/main/java/org/apache/kylin/common/util/EncryptUtil.java [hunk 1]
```diff
@@ -28,8 +30,10 @@
     /**
      * thisIsAsecretKey
      */
-    private static byte[] key = { 0x74, 0x68, 0x69, 0x73, 0x49, 0x73, 0x41, 0x53, 0x65, 0x63, 0x72, 0x65, 0x74, 0x4b,
-            0x65, 0x79 };
+    private static byte[] key = new byte[16];
+    {
```
Comment (shaofengshi) [inline core-common/src/main/java/org/apache/kylin/common/util/EncryptUtil.java]: This block won't be executed. Please fix it.

**Comment by shaofengshi** [comment]
This won't be fixed.

**Pull request closed without merging**